docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md

```diff
@@ -1,8 +1,215 @@
+x = torch.rand((10, 3, 256, 256))
+y = torch.ones((10, ), dtype=torch.long)
+ds = TensorDataset(x, y)
+dataloader = DataLoader(ds, batch_size=2)
+
+# (Optional) step 3: Something else, like training ...
+trainer = Trainer()
+trainer.fit(model, dataloader)
+...
+...
+
+# Inference/Prediction
+trainer.validate(ort_model, dataloader)
+trainer.test(ort_model, dataloader)
+trainer.predict(ort_model, dataloader)
+```
+### ONNXRuntime Acceleration
+Before you start with any BigDL-Nano usage, you are required to install some onnx packages as follow to setup your environment with ONNXRuntime acceleration.
+```shell
+pip install onnx onnxruntime
+```
+When you're ready, you can simple append following part to enable your onnxruntime acceleration.
+```python
+# step 4: trace your model as a onnxruntime model
+# if you have run `trainer.fit` before trace, then argument `input_sample` is not required.
+ort_model = Trainer.trace(model, accelerator='onnruntime', input_sample=x)
+
+# step 5: use returned model for transparent acceleration
+# The usage is almost the same with any pytorch module
+y_hat = ort_model(x)
+
+# validate, predict, test in Trainer also support acceleration
+trainer.validate(ort_model, dataloader)
+trainer.test(ort_model, dataloader)
+trainer.predict(ort_model, dataloader)
+# note that `ort_model` is not trainable any more, so you can't use like 
+# trainer.fit(ort_model, dataloader) # this is illegal
+```
+### OpenVINO Acceleration
+To use OpenVINO acceleration, you have to install openvino toolkit:
+```shell
+pip install openvino-dev
+```
+The OpenVINO usage is quite similar to ONNXRuntime, following usage is for openvino:
+```python
+# step 4: trace your model as a openvino model
+# if you have run `trainer.fit` before trace, then argument `input_sample` is not required.
+ov_model = Trainer.trace(model, accelerator='openvino', input_sample=x)
+
+# step 5: use returned model for transparent acceleration
+# The usage is almost the same with any pytorch module
+y_hat = ov_model(x)
+
+# validate, predict, test in Trainer also support acceleration
+trainer = Trainer()
+trainer.validate(ort_model, dataloader)
+trainer.test(ort_model, dataloader)
+trainer.predict(ort_model, dataloader)
+# note that `ort_model` is not trainable any more, so you can't use like 
+# trainer.fit(ort_model, dataloader) # this is illegal
+```
+
+## Quantization
+Quantization is widely used to compress models to lower precision which not only reduces the model size but also accelerates inference speed. BigDL-Nano provides `Trainer.quantize()` API for users to quickly obtain a quantized model with accuracy control by specifying a few arguments. Intel Neural Compressor (INC) and Post-training Optimization Tools (POT) from OpenVINO toolkit are enabled as options. At the meantime, runtime acceleration is also included directly in quantization pipeline when using accelerator='onnxruntime'/'openvino' so you don't have to run `Trainer.trace` before quantization.
```

Comment:
"to compress models to lower precision" -> "to compress models to a lower precision"?
"to lower precision which not only" -> "to lower precision, which not only"?
"accelerates inference speed" -> "accelerates inference" or "improve inference speed"?